### PR TITLE
chore: update meta and description

### PR DESCRIPTION
**Pull request by raisedadead**

### CODE REVIEW

File: client/src/components/landing/index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ export const Landing = ({ edges }) => {
   return (
     <Fragment>
       <Helmet>
-        <title>Learn to code | freeCodeCamp.org</title>
+        <title>Learn to code for free from home | freeCodeCamp.org</title>
```
Comment (RandellDawson) [inline client/src/components/landing/index.js]: Should it be `from home` or `at home`?

Comment (ojeytonwilliams) [reply client/src/components/landing/index.js]: 'at home' sounds more natural to me.

Comment (raisedadead) [reply client/src/components/landing/index.js]: I think I will have to confirm once again from Quincy. It depends on what people are searching on Google.

Comment (RandellDawson) [reply client/src/components/landing/index.js]: Or:
> Learn to code at home for free

There are other possibilities of course.

Comment (raisedadead) [reply client/src/components/landing/index.js]: @QuincyLarson can you confirm the string please.

Comment (QuincyLarson) [reply client/src/components/landing/index.js]: I agree with @RandellDawson and @ojeytonwilliams and Google Trends agrees as well:

_work_at_home____work_from_home____learn_from_home____learn_at_home__-_Explore_-_Google_Trends

"Learn to code for free at home" it is.

File: client/src/components/landing/index.js [hunk 2]
```diff
@@ -45,8 +45,8 @@ export const Landing = ({ edges }) => {
                 Earn certifications.
               </h2>
               <h2 className='medium-heading text-center'>
-                Since 2014, more than 40,000 freeCodeCamp.org graduates have
-                gotten jobs at tech companies including:
+                Since 2015, 40,000 graduates have gotten jobs at tech companies
+                including:
```
Comment (QuincyLarson) [inline client/src/components/landing/index.js]: I disagree with these changes. I think the original copy is stronger and more clear. Also, we started in 2014 and being older is more beneficial in this case.

Comment (raisedadead) [reply client/src/components/landing/index.js]: Then the copy on news is incorrect? A quick google search for freeCodeCamp give different results.

**Pull request merged**